editor/import/resource_importer_scene.cpp

```diff
@@ -1678,6 +1871,7 @@ void ResourceImporterScene::get_import_options(const String &p_path, List<Import
 		script_ext_hint += "*." + E;
 	}
 
+	r_options->push_back(ImportOption(PropertyInfo(Variant::BOOL, "nodes/apply_scene_root"), true));
```

Comment:
I would rename to 'apply_root_scale'

Reply:
I renamed to 'apply_root_basis'. Hope that fits.